src/commcare_cloud/ansible/roles/barman/templates/streaming-pg.conf.j2

```diff
@@ -0,0 +1,7 @@
+[{{ item }}]
+description =  "{{ item }} (Streaming-Only)"
+conninfo = "host={{ groups[item].0 }} user={{ postgres_users.barman.username }} dbname=postgres password={{ postgres_users.barman.username }}"
```

Comment:
`item` here should already be the host: https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#magic-variables-and-how-to-access-information-about-other-hosts

Reply:
I used `host` instead of `groups[item].0` but it is giving the hostname in my case its `db1`. I wanted to configure the IP address of the server here. 
I also tried using `hostvars[host]['ansible_eth0']['ipv4']['address']` but I got error `AnsibleUndefinedVariable: 'dict object' has no attribute 'ansible_eth0'` So I used the above way.

Reply:
You could try `hostvars[item]['inventory_hostname']`

Reply:
this is also giving `db1` not the IP.

Reply:
Thanks worked like charm.